ra-optimisation/linear-rao/src/main/java/com/farao_community/farao/linear_rao/config/LinearRaoParameters.java

```diff
@@ -23,13 +23,15 @@
     static final int DEFAULT_MAX_NUMBER_OF_ITERATIONS = 10;
     static final boolean DEFAULT_SECURITY_ANALYSIS_WITHOUT_RAO = false;
     static final double DEFAULT_PST_SENSITIVITY_THRESHOLD = 0.0;
+    static final double DEFAULT_PST_PENALTY_COST = 0.5;
```

Comment:
Why did you change default value (which was 1.0 previously) ?

Reply:
it is specified to be set at 0.01. I previously put 0.5 to not break cucumber tests. Now I respect the specification so I will have another PR in cucumber tests